### PR TITLE
Wrapping Podcast Promo

### LINKED ISSUE #9443: Create prototype to validate text wrapping in the CPS story page (Test only)
**Is your feature request related to a problem? Please describe.**
Create a basic prototype to check on the impact of text wrapping in the main article. This will be replaced by the podcast promo in the future

**Describe the solution you'd like**
In the CPS story page article, create a prototype to include a space with the same dimensions as the podcast promo box and to appear after the 7th paragraph. This can be for all breakpoints.  

**Describe alternatives you've considered**

**Testing notes**
Check the different test pages as documented here: https://github.com/bbc/simorgh/issues/9390

[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Checklist**
- [ ] (BBC contributors only) This issue follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Additional context**

### PR DESCRIPTION
**Pull request by ryanmccombe**
Resolves https://github.com/bbc/simorgh/issues/9443

**Overall change:**
Add a floating gray rectangle next to the 7th paragraph of STYs on TEST

Note, the positioning rules and stuff are in this PR: https://github.com/bbc/simorgh/pull/9450

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [ ] I have assigned this PR to the Simorgh project
- [ ] (BBC contributors only) This PR follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false yarn test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by DarioR01 — CHANGES_REQUESTED** [review]
I think this should appear after the 7th paragraph, and it should not appear at all if there are less than 8 paragraphs.
Screenshot 2021-09-23 at 11 12 13
.

**Comment by DarioR01** [comment]
> I think this should appear after the 7th paragraph, and it should not appear at all if there are less than 8 paragraphs.
> Screenshot 2021-09-23 at 11 12 13
> .

Before I keep on reviewing it, is this meant to be a prototype that includes a grey box without any logic on when and where it should appear?
As from the logic described in https://github.com/bbc/simorgh/issues/9390

**Comment by ryanmccombe** [comment]
> As from the logic described in #9390

That's a different issue 🙂  This PR is for https://github.com/bbc/simorgh/issues/9443

I'm working on a different PR for https://github.com/bbc/simorgh/issues/9390 (https://github.com/bbc/simorgh/pull/9450)

**Review by DarioR01 — APPROVED** [review]
It makes sense :D. Then I have nothing to say other than approving it :D

**Review by rebeccamcginn — APPROVED** [review]

File: src/app/containers/PodcastPromo/inline.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+const Prototype = styled.div`
+  background: #f4f4f4;
+  float: left;
+
+  width: 115px;
+  height: 370px;
+  margin: 0 6px 6px 8px;
+
+  @media (min-width: 320px) {
```
Comment (amoore108) [inline src/app/containers/PodcastPromo/inline.jsx:14]: Should these use the `@bbc/gel-foundations/breakpoints` values? Probably not really needed for prototype purposes mind!

Comment (ryanmccombe) [reply src/app/containers/PodcastPromo/inline.jsx:14]: Yep definitely - work seems to have been broken up into 4 issues though (the last 4 in this milestone: https://github.com/bbc/simorgh/milestone/128).  Implementing the final design will be one of those.

This is just a box of the rough size so designs / placement rules can be refined after seeing how it looks with real data I think

Comment (amoore108) [reply src/app/containers/PodcastPromo/inline.jsx:14]: Ah cool, looks good to me!

**Review by amoore108 — APPROVED** [review]

**Pull request merged**